contracts/protocol/facets/OfferHandlerFacet.sol

```diff
@@ -141,13 +141,7 @@ contract OfferHandlerFacet is IBosonOfferHandler, OfferBase {
      * @param _offerId - the id of the offer
      * @param _length - the length of the range
      */
-    function reserveRange(uint256 _offerId, uint256 _length)
-        external
-        override
-        nonReentrant
```

Comment:
My previous comment is related to this `nonReenterant` modifier

Reply:
`nonReentrant` is still here. Prettier just put it into the same line with function name and other stuff.